src/containers/Firebase/firebase.js

```diff
@@ -41,6 +41,12 @@ class Firebase {
   user = uid => this.db.doc(`users/${uid}`);
 
   users = () => this.db.collection('users');
+
+  // create collection for journals
+  // and each journal will be a document
+  journal = Jid => this.db.doc(`journals/${Jid}`);
```

Comment:
Write descriptive variables names 
```suggestion
  journal = journalId => this.db.doc(`journals/${journalId}`);
```